src/api/imageUploadApi.ts

```diff
@@ -0,0 +1,40 @@
+import { axiosInstance } from './axios';
+import { VITE_SERVER_URL } from '../utils/importEnvVariable';
+import { typeGuard } from '../utils/typeGuard';
+
+export interface IPresignedImgUrl {
+  key: string;
+  presigned: {
+    fields: {
+      bucket: string;
+      key: string;
+      Policy: string;
+      'X-Amz-Algorithm': string;
+      'X-Amz-Credential': string;
+      'X-Amz-Date': string;
+      'X-Amz-Signature': string;
+    };
```

Comment:
명확해서 좋긴한데, 혹시라도 변경될수도 있는부분이라
```suggestion
    fields: {
      [key: string]: string;
    };
```

으로 해도 괜찮을거같아유

Reply:
오 훨씬 좋네요. 감사합니다!